### PR TITLE
Add better Grafana logging to Toriel

### PR DESCRIPTION
**Pull request by aboutdavid**
This is a draft, but here's what it does so far:

- Creates a new alert utility file, exposing an info and an urgent. Info logs to Slack, while Urgent does Slack
- Starts adding more Grafana tracking to almost everything

### CODE REVIEW

**Comment by jaspermayone** [comment]
@aboutdavid we should not just use increment metrics, but also track response times to certain actions. (user invite, etc)

**Comment by aboutdavid** [comment]
> @aboutdavid we should not just use increment metrics, but also track response times to certain actions. (user invite, etc)

okay, will add timers and other stuff

File: interactions/cleanup-cave.js [hunk 1]
```diff
@@ -14,10 +15,20 @@ async function cleanupCaveChannel(dryRun = true) {
   })
 
   if (dryRun) {
```
Comment (tregsthedev) [inline interactions/cleanup-cave.js:17]: Where is this logging?

File: util/alert.js
Comment (tregsthedev) [inline util/alert.js:1]: Is this working?

**Comment by jaspermayone** [comment]
@aboutdavid is this ready for my review?

**Review by jaspermayone — APPROVED** [review]

**Pull request merged**